app/models/katello/concerns/smart_proxy_extensions.rb

```diff
@@ -39,9 +39,10 @@ module SmartProxyExtensions
                  :through    => :capsule_lifecycle_environments,
                  :source     => :lifecycle_environment
 
-        has_many :hosts,      :class_name => "::Host::Managed", :foreign_key => :content_source_id,
-                              :inverse_of => :content_source
-        has_many :hostgroups, :class_name => "::Hostgroup",     :foreign_key => :content_source_id,
+        has_many :content_facets, :class_name => "::Katello::Host::ContentFacet", :foreign_key => :content_source_id,
+                                  :inverse_of => :content_source
```

Comment:
In content_facet.rb, do you also need to update the 'inverse_of', since changing this from 'hosts' to 'content_facets'?

e.g. current content_facet.rb:
```
      belongs_to :content_source, :class_name => "::SmartProxy", :foreign_key => :content_source_id, :inverse_of => :hosts
```